### ISSUE TITLE
test: buildbot.test.integration.test_telegram_bot is flaky

### ISSUE THREAD

**Issue by p12tic** [buildbot/buildbot#5120]
Fails due to database errors:
```
[ERROR]
Traceback (most recent call last):
  File "/buildbot/buildbot-job/build/master/buildbot/reporters/words.py", line 1057, in _load_channels_state
    data = yield self.master.db.state.getState(objectid, attr, ())
  File "/buildbot/buildbot-job/build/sandbox/lib/python3.7/site-packages/twisted/internet/defer.py", line 1416, in _inlineCallbacks
    result = result.throwExceptionIntoGenerator(g)
  File "/buildbot/buildbot-job/build/sandbox/lib/python3.7/site-packages/twisted/python/failure.py", line 512, in throwExceptionIntoGenerator
    return g.throw(self.type, self.value, self.tb)
  File "/buildbot/buildbot-job/build/master/buildbot/db/pool.py", line 229, in do
    args, kwargs)
  File "/buildbot/buildbot-job/build/sandbox/lib/python3.7/site-packages/twisted/python/threadpool.py", line 250, in inContext
    result = inContext.theWork()
  File "/buildbot/buildbot-job/build/sandbox/lib/python3.7/site-packages/twisted/python/threadpool.py", line 266, in <lambda>
    inContext.theWork = lambda: context.call(ctx, func, *args, **kw)
  File "/buildbot/buildbot-job/build/sandbox/lib/python3.7/site-packages/twisted/python/context.py", line 122, in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
  File "/buildbot/buildbot-job/build/sandbox/lib/python3.7/site-packages/twisted/python/context.py", line 85, in callWithContext
    return func(*args,**kw)
  File "/buildbot/buildbot-job/build/master/buildbot/db/pool.py", line 190, in __thd
    rv = callable(arg, *args, **kwargs)
  File "/buildbot/buildbot-job/build/master/buildbot/db/state.py", line 98, in thd
    return self.thdGetState(conn, objectid, name, default=default)
  File "/buildbot/buildbot-job/build/master/buildbot/db/state.py", line 107, in thdGetState
    res = conn.execute(q)
  File "/buildbot/buildbot-job/build/sandbox/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 988, in execute
    return meth(self, multiparams, params)
  File "/buildbot/buildbot-job/build/sandbox/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 287, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/buildbot/buildbot-job/build/sandbox/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1107, in _execute_clauseelement
    distilled_params,
  File "/buildbot/buildbot-job/build/sandbox/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1253, in _execute_context
    e, statement, parameters, cursor, context
  File "/buildbot/buildbot-job/build/sandbox/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1473, in _handle_dbapi_exception
    util.raise_from_cause(sqlalchemy_exception, exc_info)
  File "/buildbot/buildbot-job/build/sandbox/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 398, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/buildbot/buildbot-job/build/sandbox/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1249, in _execute_context
    cursor, statement, parameters, context
  File "/buildbot/buildbot-job/build/sandbox/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 580, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such table: object_state
[SQL: SELECT object_state.value_json 
FROM object_state 
WHERE object_state.objectid = ? AND object_state.name = ?]
[parameters: (2, 'missing_workers')]
(Background on this error at: http://sqlalche.me/e/e3q8)
```

https://buildbot.buildbot.net/#/builders/23/builds/1559/steps/8/logs/stdio

https://buildbot.buildbot.net/#/builders/59/builds/86/steps/10/logs/stdio

https://buildbot.buildbot.net/#/builders/66/builds/81/steps/8/logs/stdio

**Comment by rjarry** [comment]
Could this be fixed by commit 81d693a8e973b51552cd23bfa695c4b2b0a30f2c ?

**Comment by p12tic** [comment]
It could be that these were the final part needed because I can no longer reproduce the issues.
